scripts/indexScenes.ts

```diff
@@ -0,0 +1,86 @@
+import * as fs from 'fs'
+import * as path from 'path'
+
+import * as crypto from '../src/crypto'
```

Comment:
If you want to be consistent with the import rules from the standards, these should look like:

```

import * as fs from 'fs'
import * as path from 'path'
import { Log, env } from 'decentraland-commons'
import { db, Model } from 'decentraland-server'

import * as crypto from '../src/crypto'
```

super minor tho